web/assets/main.scss

```diff
@@ -26,3 +26,7 @@ $moj-page-width: 1220px;
 .table-row__no-border {
    border-bottom: none;
  }
+
+.colour-text-red {
+   color: govuk-colour("red") !important;
```

Comment:
Is there another way to override styles without using `!important` which I know should be avoided. Without it, the style was being overridden by `.govuk-link--no-visited-state:visited`

Reply:
I think it's fine in this context but I've added a comment about naming it to be a bit clearer.

An alternative would be updating the selector to also target pseudoselectors:
```.colour-text-red, .colour-text-red:visited {```
But I suspect you'd quickly find you need to do all of them:
```.colour-text-red, .colour-text-red:visited, .colour-text-red:active, .colour-text-red:hover {```